src/diffusers/models/adapter.py

```diff
@@ -399,7 +409,7 @@ def __init__(self, in_channels: int, out_channels: int, num_res_blocks: int, dow
 
         self.downsample = None
         if down:
-            self.downsample = Downsample2D(in_channels)
+            self.downsample = nn.AvgPool2d(kernel_size=2, stride=2, ceil_mode=True)
```

Comment:
Note to reviewers:

`Downsample2D(in_channels)` is really just a `nn.AvgPool2d(kernel_size=2, stride=2)` layer. The only real change here is setting `ceil_mode=True`.